### PR TITLE
add new command to telegram bot

### PR DESCRIPTION
**Pull request by BourgeoisBenjamin**
- added possibility to change command name in config.json
- added /config to show current configuration
- added /show_unchanged, /show_decrease, /show_decrease_to_zero, /show_increase, /show_increase_from_zero to change configuration
- set list of available commands in telegram chat when bot started
- new emoji in telegram message :)

### CODE REVIEW

**Comment by marklagendijk** [comment]
Nice work! Thanks for the PR!

File: config.defaults.json [hunk 1]
```diff
@@ -22,7 +22,41 @@
     "telegram": {
       "enabled": false,
       "botToken": "See README",
-      "chats": []
+      "chats": [],
+      "commands": {
```
Comment (marklagendijk) [inline config.defaults.json:26]: I don't see what it adds to make these configurable.
Since I like to keep things simple, I would remove this config.

Comment (BourgeoisBenjamin) [reply config.defaults.json:26]: I hesitated with that. I use this part for the getCommands function used to set commands list in Telegram chat but I can move it into telegram-bot.js

Comment (marklagendijk) [reply config.defaults.json:26]: Please do

Comment (BourgeoisBenjamin) [reply config.defaults.json]: done ;)

**Comment by BourgeoisBenjamin** [comment]
updated!

**Comment by marklagendijk** [comment]
Merged and released!

**Pull request merged**